modules/ROOT/pages/prereq-firewalld-forwarding.adoc

```diff
@@ -0,0 +1,46 @@
+= Enable Forwarding When Using firewalld
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+`firewalld` is an iptables controller that defines rules for persistent network traffic.
+If you are using `firewalld` with a Red Hat Enterprise Linux (RHEL) 7.3 operating system, you must enable forwarding on the docker0 device.
+You must also forward any packets being sent from or to the 10.0.0.0/8 subnet.
+
+== To Determine if You Are Using firewalld
+
+To determine if your system is using firewall run the following command:
```

Comment:
firewall, run

Reply:
Got it - thank you!

Reply:
This should be firewalld -- not a typo :)

Reply:
Got it - thanks!